event_routing_backends/processors/xapi/event_transformers/problem_interaction_events.py

```diff
@@ -274,18 +276,19 @@ def get_result(self):
             Result
         """
         # Do not transform result if the event is generated from browser
-        if self.event['context']['event_source'] == 'browser':
+        if self.get_data('context.event_source') == 'browser':
             return None
 
-        event_data = self.event['data']
+        event_data = self.get_data('data', True)
 
         return Result(
-            success=event_data['success'] == 'correct',
+            success=event_data['success'] == 'correct' if 'success' in event_data else False,
```

Comment:
Instead of `if/else` statements we should use get method of dict to fallback to a default value i.e 
`event_data.get('success', None)`

Reply:
DONE